about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles.css" />
+    <link rel="stylesheet" href="/mobile.css" />
+    <script src="index.js" defer></script>
+
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="dark-bar">
+      <nav class="container dark-nav">
```

Comment:
[ OPTIONAL ] Social icons on the top nav bar should have the same design and font color from the design.
![socials](https://user-images.githubusercontent.com/35504149/137521818-dcf19fd5-b913-4946-bf77-f38bc06cc73c.png)